src/components/CreateNewEvent.js

```diff
@@ -218,7 +211,7 @@ function CreateNewEvent() {
   const handleSpeakerChange = (e) => {
     const { id, name, value } = e.target;
     const s = [...speaker];
-    s[parseInt(id, 10)][name] = value;
+    s[parseInt(id)][name] = value;
```

Comment:
missing radix parameter take reference [here](https://stackoverflow.com/questions/7818903/jslint-says-missing-radix-parameter)

Reply:
This all errors are not from my code someone has added these functions i don't know who is but that person has not resolved all this and i have to face this all issue,